packages/web-components/src/components/callout/callout.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { customElement, html, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
+import styles from './callout.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Link list.
+ *
+ * @element dds-link-list
+ */
+@customElement(`${ddsPrefix}-callout`)
+class DDSCallout extends LitElement {
+  static styles = styles; // `styles` here is a `CSSResult` generated by custom WebPack loader
+
+  static get stableSelector() {
+    return `${ddsPrefix}--callout`;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  protected render() {
```

Comment:
To align to the rest of the codebase:

```suggestion
  render() {
```

Reply:
I've updated it, thanks for the review!